### PR TITLE
Samples-WindowsDX.sln updated

### PR DESCRIPTION
**Pull request by ghost**
I've updated Samples-WindowsDX.sln, Samples-WindowsGL.sln and Samples-Android.sln to build in VS 2019 using MonoGame 3.7.1.189. The Windows samples were all converted to dotnet SDK-style project files, and "dotnet build" works as well. Added WindowsDX build for NeonShooter and SpaceWar. Built and tested each project.

### CODE REVIEW

File: Platformer2D/Platforms/Android/Platformer2D.csproj [hunk 1]
```diff
@@ -12,20 +12,21 @@
...
     <AndroidManifest>Properties\AndroidManifest.xml</AndroidManifest>
+    <AndroidEnableSGenConcurrent>true</AndroidEnableSGenConcurrent>
+    <AndroidUseAapt2>true</AndroidUseAapt2>
+    <AndroidHttpClientHandlerType>Xamarin.Android.Net.AndroidClientHandler</AndroidHttpClientHandlerType>
     <JavaMaximumHeapSize />
     <JavaOptions />
     <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\..\..\</SolutionDir>
     <RestorePackages>true</RestorePackages>
-    <NuGetPackageImportStamp>f7f092e7</NuGetPackageImportStamp>
-    <MonoGamePlatform>Android</MonoGamePlatform>
```
Comment (SimonDarksideJ) [inline Platformer2D/Platforms/Android/Platformer2D.csproj:28]: Are we sure this line should be removed?  This was provided as part of the MonoGame platform setup to identify the platform

File: Platformer2D/Platforms/Android/Properties/AndroidManifest.xml [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android">
-	<uses-sdk android:minSdkVersion="12" android:targetSdkVersion="12" />
-	<application></application>
-</manifest>
\ No newline at end of file
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="1" android:versionName="1.0" package="net.monogame.samples.platformer2d">
+	<uses-sdk android:minSdkVersion="21" android:targetSdkVersion="28" />
+	<application android:label="MonoGame.Samples.Platformer2D.Android"></application>
+        <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
```
Comment (SimonDarksideJ) [inline Platformer2D/Platforms/Android/Properties/AndroidManifest.xml]: Why would we need Network Access permissions for a sample that is not using networking. 

Also, why increase the API version requirements, which limits the devices this sample can be deployed on

**Review by SimonDarksideJ — COMMENTED** [review]
A few queries, specifically around some changes that seem unnecessary.  These are usually the result of compiling a platform with out using a MonoGame template.

File: Platformer2D/Platforms/Android/Platformer2D.csproj (hunk 1 above)
Comment (ghost) [reply Platformer2D/Platforms/Android/Platformer2D.csproj:28]: I believe the NuGetPackageImportStamp is a holdover of the pre-PackageReference world. It was used in the packages.config old style. I don't believe it is necessary anymore. In fairness, I'm not 100% sure of the significance but I tested clean builds and I know it is not strictly speaking necessary.

The MonoGamePlatform=Android specification is definitely not necessary. The reason is the .targets file in the MonoGame.Framework.Android nuget package already specifies that.

File: Platformer2D/Platforms/Android/Properties/AndroidManifest.xml (hunk 2 above)
Comment (ghost) [reply Platformer2D/Platforms/Android/Properties/AndroidManifest.xml]: I believe both of these changes are some kind of default coming in from an updated android SDK. I'll try to fix both of these.

Comment (ghost) [reply Platformer2D/Platforms/Android/Properties/AndroidManifest.xml]: This is fixed now. I tested clean build and ran it on the emulator (Pixel XL, apiversion=29).

File: NeonShooter/Game.cs [hunk 3]
```diff
@@ -42,10 +42,9 @@ public NeonShooterGame()
         {
 			Instance = this;
 			graphics = new GraphicsDeviceManager(this);
-			Content.RootDirectory = "Content";
 
-            graphics.PreferredBackBufferWidth = 1920;
-            graphics.PreferredBackBufferHeight = 1080;
+            //graphics.PreferredBackBufferWidth = 1920;
```
Comment (SimonDarksideJ) [inline NeonShooter/Game.cs:46]: Can we please remove lines if not used. Shouldn't have commented out code in samples

**Review by SimonDarksideJ — CHANGES_REQUESTED** [review]
Last changes and then this should be good to go.

Although, the project will need updating to the upcoming 3.8 release due soon

**Review by SimonDarksideJ — APPROVED** [review]

**Pull request merged**